test/system/090-events.bats

```diff
@@ -291,3 +291,21 @@ EOF
     _events_container_create_inspect_data journald
     _events_container_create_inspect_data file
 }
+
+@test "event with too big labels" {
+    # test that events entries are not truncated when reading from syslog
+
+    cname=test-$(random_string 30 | tr A-Z a-z)
+
+    local label_file="$BATS_TEST_TMPDIR/labels.txt"
+    echo -n "too-long-label=" >> "$label_file"
+    random_string 150000 >> "$label_file"
+
+    local start="$(date --iso-8601=ns)"
```

Comment:
The man page says that rfc-3339 (2022-12-08 18:23:31.758649447+01:00) should be used,
but that doesn't work.
The iso format works (2022-12-08T18:24:25,699552059+01:00).
Why?

Reply:
Either Go, or the `date` util have got some formats wrong.

Reply:
Probably GNU date bug?

Reply:
Sorry, what's the question? If the man page says 4rfc3339, my first guess would be that the person who wrote it wasn't 100% clear on the differences between rfc3339 and iso8601. (Which is fine, because I've been dealing with time representations for thirty years and still have trouble with it.) Since iso8601 works, I'd suggest fixing the man page.

Reply:
The issues is that GNU `date` util separates date & time by a space, not by capital T.
The iso8601 format uses `T` as it should but it uses comma `,` as decimal separator so it is wrong too.
I have given up and now I am using `--since=10s`.

Reply:
That's unfortunate, and it's very clearly a bug: its own documentation states that it uses a period. Seems worth reporting to bug-coreutils@gnu.org.

Reply:
from wiki
>A decimal mark, either a comma or a dot is used as a separator between the time element and its fraction. (Following ISO 80000-1 according to ISO 8601:1-2019,[26] it does not stipulate a preference except within International Standards, but with a preference for a comma according to ISO 8601:2004.